doc/source/whatsnew/v1.1.4.rst

```diff
@@ -33,8 +33,9 @@ Bug fixes
 
 .. _whatsnew_114.other:
 
-Other
-~~~~~
+Indexing
+~~~~~~~~
+- Bug in :meth:`Series.isin` and :meth:`DataFrame.isin` raising a ``ValueError`` when the target was read-only (:issue:`37174`)
```

Comment:
hmm. why a separate section? Why not just in bug fixes? This is what was done in 1.1.1, 1.1.2 and 1.1.3.

Reply:
Fixed